.changeset/yellow-glasses-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': patch
+---
+
+fetch project versions in view mode
```

Comment:
Can you capitalize the first character of this sentence as well as giving some idea why we need to do this here and in the summary of the PR?